acceptance/tests/load_libfacter.rb

```diff
@@ -1,31 +1,32 @@
-test_name 'Ruby can load libfacter'
+test_name 'C100161: Ruby can load libfacter' do
+  tag 'risk:high'
 
-agents.each do |agent|
-  on agent, "env PATH=\"#{agent['privatebindir']}:${PATH}\" which ruby"
-  ruby_path = stdout.chomp
+  agents.each do |agent|
+    ruby_path = on(agent, "env PATH=\"#{agent['privatebindir']}:${PATH}\" which ruby").stdout.chomp
```

Comment:
I looked at using ruby_command, but based on what the test appears to be doing it doesn't really work. I've added comments explaining what its doing and why